.github/workflows/claude-nl-suite.yml

```
@@ -1,134 +1,463 @@
-      - name: Install UnityMcpServer deps
-        run: |
-          set -eux
-          if [ -f "UnityMcpBridge/UnityMcpServer~/src/pyproject.toml" ]; then
-
-            uv venv
-            . .venv/bin/activate
-            uv pip install -e "UnityMcpBridge/UnityMcpServer~/src"
-          elif [ -f "UnityMcpBridge/UnityMcpServer~/src/requirements.txt" ]; then
+name: Claude NL/T Suite (Unity live)
+  
+  on:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 4: mapping values are not allowed in this context

(syntax-check)